ansible/environments/docker-machine/group_vars/all

```diff
@@ -0,0 +1,35 @@
+whisk_version_name: mac
+nginx_conf_dir: /Users/Shared/nginx
+consul_conf_dir: /Users/Shared/consul
+whisk_logs_dir: /Users/Shared/wsklogs
+cli_nginx_dir: "{{ nginx_conf_dir }}/cli/go/download"
+docker_registry: ""
+docker_dns: ""
+
+db_prefix: "{{ ansible_user_id }}_{{ ansible_hostname|lower }}_"
```

Comment:
Bot seeing the update to db_prefix to assure ansible_user_id is lower cased:
https://github.com/apache/incubator-openwhisk/pull/2217